### DISCUSSION TITLE
flask: error: unrecognized arguments: db init

### DISCUSSION THREAD

**Discussion by nguyenminhan** [miguelgrinberg/Flask-Migrate#422] [Q&A]
help me !
usage: flask [-h] [--trained_model TRAINED_MODEL] [--text_threshold TEXT_THRESHOLD] [--low_text LOW_TEXT]
             [--link_threshold LINK_THRESHOLD] [--canvas_size CANVAS_SIZE] [--mag_ratio MAG_RATIO] [--poly]
             [--show_time] [--refine] [--refiner_model REFINER_MODEL]
flask: error: unrecognized arguments: db init

**Comment by miguelgrinberg** [comment]
Have you installed Flask-Migrate?
